addons/website/static/src/js/menu/edit.js

```diff
@@ -158,6 +172,12 @@ var EditPageMenu = websiteNavbarData.WebsiteNavbarActionWidget.extend({
             editableMode: true,
             $target: ev.data.$target,
         });
+        if (ev.data.$origin) {
```

Comment:
Not sure why this `if` is useful ?

Reply:
As discussed, it is to prevent stable possible custom apps to suddently call this method which will have a new parameter.
If the parameter is not passed, it will reload all #wrapwrap animations.
Safety net for stable, to remove on master as for Odoo standard code we always pass $origin with this PR.